### PR TITLE
Various code cleanup around test code

### PR DESCRIPTION
**Pull request by Geod24**
The title says it all. Review commit-by-commit to better see what's happening there.

### CODE REVIEW

File: lib/sync/fetcher_test.go [hunk 1]
```diff
@@ -36,9 +35,9 @@ func TestBlockFetcher(t *testing.T) {
 	st := block.InitTestBlockchain()
 	defer st.Close()
 
-	kp := keypair.Random()
-	_, _, localNode := network.CreateMemoryNetwork(nil)
+	localNode := node.NewTestLocalNode0()
 
+	kp := keypair.Random()
```
Comment (soonkuk) [inline lib/sync/fetcher_test.go]: Why did you remove from above line and inserted again?

Comment (Geod24) [reply lib/sync/fetcher_test.go]: I just grouped it with its usage (`NewValidator`). Because a local node also takes a keypair.
